web/modules/custom/bnf/bnf_client/src/Plugin/QueueWorker/NodeUpdate.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Drupal\bnf_client\Plugin\QueueWorker;
+
+use Drupal\autowire_plugin_trait\AutowirePluginTrait;
+use Drupal\bnf\Services\BnfImporter;
+use Drupal\bnf_client\Form\SettingsForm;
+use Drupal\Core\Config\ConfigFactoryInterface;
+use Drupal\Core\Plugin\ContainerFactoryPluginInterface;
+use Drupal\Core\Queue\QueueWorkerBase;
+use Psr\Log\LoggerInterface;
+use Symfony\Component\DependencyInjection\Attribute\Autowire;
+
+/**
+ * Update node content.
+ *
+ * @QueueWorker(
+ *   id = "bnf_client_node_update",
+ *   title = @Translation("Update node content."),
+ *   cron = {"time" = 60}
```

Comment:
Would it be silly to add a notice that this is in minutes, and not seconds?

Reply:
No, because it is, and I don't think it means what you think it does.

It's the queue item lease time, it's how long the cron worker reserves the item when starting to work on it to stop anyone else from claiming it. Per default it's 15 seconds, but, for some reason, every `QueueWorker` example you Google uses 60 seconds.

While 15 seconds should generally be enough, 60 seems a bit safer, in case something gets bogged down.